AnalysisBasics_details.md

```diff
@@ -0,0 +1,23 @@
+## Basics
```

Comment:
This section does indeed overlap with what's in Python Development. I agree with Claire's suggestion to put Code Development and Management first, since it deals with coding fundamentals. If so, I would say this section should be taken out of here and blended with what's there in Python Development and Python Details.